industrial_ci/ci_main.sh

```diff
@@ -325,4 +325,4 @@ cd $TARGET_REPO_PATH  # cd back to the repository's home directory with travis
 pwd
 
 HIT_ENDOFSCRIPT=true
-success 0
```

Comment:
I had similar issues in my GitLab script, you never want to call `exit` but instead `return` if you source scripts!